optimas/generators/base.py

```diff
@@ -250,7 +257,11 @@ def tell(
             incorporating the evaluated trials. By default ``True``.
 
         """
-        self._tell(trials)
+        # Feed data to the generator, using the standardized API
+        self.ignore_trials(trials)
+        self.ingest([trial.to_dict() for trial in trials])
```

Comment:
Should we only call in ingest when the trial evaluation was successful?

Is currently called even when the evaluation has failed, which results in `KeyError` In the  libEnsemble generator.
Evaluation parameters are not mapped to a trial unless the trial is successful.

Reply:
We might also want to warn when there are fail trials